build-tools/src/main/kotlin/org/jetbrains/kotlin/CopySamples.kt

```diff
@@ -34,6 +32,22 @@ open class CopySamples: Copy() {
                 return@filter line
             }
         }
+        from(samplesDir) {
+            it.include("**/gradle.properties")
+
+            val kotlinVersion = project.property("kotlinVersion") as? String
+                ?: throw IllegalArgumentException("Property kotlinVersion should be specified in the root project")
+            val kotlinCompilerRepo = project.property("kotlinCompilerRepo") as? String
+                ?: throw IllegalArgumentException("Property kotlinCompilerRepo should be specified in the root project")
+
+            it.filter { line ->
+                when {
+                    line.contains("kotlin_version") ->  "kotlin_version=$kotlinVersion"
```

Comment:
(minor) The "kotlin_version" substring may be a part of a comment, so may be use `startsWith` instead of `contains`? The same for "kotlinCompilerRepo" below.